passing-args-to-yield/solution.md

```diff
@@ -5,16 +5,14 @@
 For comparison, here is a possible solution, so you can compare notes:
 
 ```js
-module.exports = function *generate() {
+module.exports = function *multiplier() {
   var num = 0,
-      multiplier = 1;
+      mul = 1;
```

Comment:
Do you mind changing it to `mult` - this makes it visually distinct from `num`, increasing the readability of this code overall :+1: